TwitchPlaysAssembly/Src/Helpers/UrlHelper.cs

```diff
@@ -28,6 +28,8 @@ public static bool ToggleMode()
 
 	public static string ManualFor(string module, string type = "html", bool useVanillaRuleModifier = false) => string.Format(TwitchPlaySettings.data.RepositoryUrl + "{0}/{1}.{2}{3}", type.ToUpper(), NameToUrl(module), type, (useVanillaRuleModifier && type.Equals("html")) ? $"#{VanillaRuleModifier.GetRuleSeed()}" : "");
 
+	public static string MissionLink(string mission) => "https://bombs.samfun.dev/mission/" + NameToUrl(mission);
+	
```

Comment:
There's some modifications that need to be done to names for the website which don't apply, so let's make a separate method just escapes a url for twitch.
```suggestion
	public static string MissionLink(string mission) => "https://bombs.samfun.dev/mission/" + TwitchUrlEscape(mission);
	
	private static string TwitchUrlEscape(string name) => Uri.EscapeDataString(Uri.UnescapeDataString(name)).Replace("*", "%2A").Replace("!", "%21");
```